shell/common/gin_converters/blink_converter.cc

```diff
@@ -292,21 +293,21 @@ bool Converter<blink::WebSize>::FromV8(v8::Isolate* isolate,
   return dict.Get("width", &out->width) && dict.Get("height", &out->height);
 }
 
-bool Converter<blink::WebDeviceEmulationParams>::FromV8(
+bool Converter<blink::DeviceEmulationParams>::FromV8(
     v8::Isolate* isolate,
     v8::Local<v8::Value> val,
-    blink::WebDeviceEmulationParams* out) {
+    blink::DeviceEmulationParams* out) {
   gin_helper::Dictionary dict;
   if (!ConvertFromV8(isolate, val, &dict))
     return false;
 
-  std::string screen_position;
-  if (dict.Get("screenPosition", &screen_position)) {
-    screen_position = base::ToLowerASCII(screen_position);
-    if (screen_position == "mobile")
-      out->screen_position = blink::WebDeviceEmulationParams::kMobile;
-    else if (screen_position == "desktop")
-      out->screen_position = blink::WebDeviceEmulationParams::kDesktop;
+  std::string screen_type;
+  if (dict.Get("screenPosition", &screen_type)) {
```

Comment:
"screen position" is a weird name for this, but probably not worth it to change it.